docs/integrations/emr/networking.md

```diff
@@ -0,0 +1,99 @@
+    </a>
+    <figcaption>Identifying the EMR VPC</figcaption>
+  </figure>
+</p>
+
+!!! info "Hopsworks installer"
+    If you are performing an installation using the [Hopsworks installer script](https://hopsworks.readthedocs.io/en/stable/getting_started/installation_guide/platforms/hopsworks-installer.html), ensure that the machines you are going to install Hopsworks on are configured with the respective VPC.
+
+!!! info "Hopsworks.ai"
+    If you are on **Hopsworks.ai**, you can directly deploy Hopsworks to the EMR VPC, by simply selecting it at the [VPC selection step during cluster creation](../../hopsworksai/aws/cluster_creation.md#step-6-vpc-selection).
+
+**Option 2: Setting up VPC peering**
+
+Follow the guide [VPC Peering](https://docs.aws.amazon.com/vpc/latest/peering/create-vpc-peering-connection.html) to set up VPC peering between the Feature Store and EMR. Get your Feature Store *VPC ID* and *CIDR* by searching for the Feature Store VPC in the AWS Management Console:
+
+!!! info "Hopsworks.ai"
+    On **Hopsworks.ai**, the VPC is shown in the cluster details.
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/databricks/aws/hopsworks_vpc.png">
+      <img src="../../../assets/images/databricks/aws/hopsworks_vpc.png" alt="Identifying the Feature Store VPC">
+    </a>
+    <figcaption>Identifying the Feature Store VPC</figcaption>
+  </figure>
+</p>
+
+## Step 2: Configuring the Security Group
+
+The Feature Store *Security Group* needs to be configured to allow traffic from your EMR clusters to be able to connect to the Feature Store.
+
+!!! note "Hopsworks.ai"
+    If you deployed your Hopsworks Feature Store instance with Hopsworks.ai, it suffices to enable [outside access of the Feature Store and Online Feature Store services](../../../hopsworksai/aws/getting_started/#step-5-outside-access-to-the-feature-store).
+
+Open your feature store instance under EC2 in the AWS Management Console and ensure that ports *443*, *9083*, *9085*, *8020* and *50010* are reachable
+from the EMR Security Group:
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/databricks/aws/databricks_security_group_overview.png">
+      <img src="../../../assets/images/databricks/aws/databricks_security_group_overview.png" alt="Hopsworks Feature Store Security Group">
+    </a>
+    <figcaption>Hopsworks Feature Store Security Group</figcaption>
+  </figure>
+</p>
+
+Connectivity from the EMR Security Group can be allowed by opening the Security Group, adding a port to the Inbound rules and setting the EMR master and core security group as source:
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/databricks/aws/databricks_security_group_details.png">
+      <img src="../../../assets/images/databricks/aws/databricks_security_group_details.png" alt="Hopsworks Feature Store Security Group details">
+    </a>
+    <figcaption>Hopsworks Feature Store Security Group details</figcaption>
```

Comment:
.

Reply:
@jimdowling I fixed most of your comments and tried to replicate them to the original sources. Not sure about the captions as we would need to fix it through the entire doc. Also the secrets manager isn't optional for EMR as spreading the API key everywhere would be very messy and I don't want to encourage people to do it. It's much worse than simply having it in your notebook as it will be put in logs all over the place.

Reply:
I am ok with removing the optional in the title, but please keep the first line describing the option of using the api key value or file. It is a big friction point for new users to set up a managed secret when they have never done it before. All they want to do is try out the feature store - the easiest way is to dump the api key in their program. For a test/transient cluster and with api key scopes - most users i expect will prefer that.

Reply:
@jimdowling You are confusing some things here. The API Key here is not used for the HSFS Api, it's needed for creating and configuring EMR clusters.

Reply:
And I'm convinced that whoever manages to configure EMR clusters the way needed won't have an issue setting up a secret, the API users can then use whatever option they want, I just link to the connect API in the end.

Reply:
Ok,i thought it was our API key. Approving.